diff --git a/node_modules/@react-google-maps/api/dist/index.d.ts b/node_modules/@react-google-maps/api/dist/index.d.ts
index 074a7b8..17197f7 100644
--- a/node_modules/@react-google-maps/api/dist/index.d.ts
+++ b/node_modules/@react-google-maps/api/dist/index.d.ts
@@ -375,7 +375,7 @@ interface ClustererState {
     markerClusterer: Clusterer | null;
 }
 interface MarkerClustererProps {
-    children: (markerClusterer: Clusterer) => JSX.Element;
+    children: (markerClusterer: Clusterer) => JSX.Element[] | JSX.Element;
     options?: ClustererOptions | undefined;
     /** Whether the position of a cluster marker should be the average position of all markers in the cluster. If set to false, the cluster marker is positioned at the location of the first marker added to the cluster. The default value is false. */
     averageCenter?: boolean | undefined;
